### PR TITLE
Update dependencies + fix some linter warnings

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Update design system packages (cfpb-) to `0.1.0`
- Update `core-js` to `3.6.5 `
- Update `@babel/core` to `7.11.4`
- Update `@babel/preset-env` to `7.11.0`
- Update `autoprefixer` to `9.8.6`
- Update `babel-jest` to `26.3.0`
- Update `babel-loader` to `8.1.0`
- Update `browser-sync` to `2.26.12`
- Update `glob` to `7.1.6`
- Update `gulp-rename` to `2.0.0`
- Update `jest` to `26.4.2`
- Update `jest-cli` to `26.4.2`
- Update `minimist` to `1.2.5`
- Update `request` to `2.88.2`
- Update `sauce-connect-launcher` to `1.3.2`
- Update `terser-webpack-plugin` to `4.1.0`
- Update `webpack` to `4.44.1`
- Update `through2` to `4.0.2`
- Move browserslist to package.json
- Switch to GH Actions from Travis.
- Fix some linter warnings.

## Testing

- `./setup.sh && gulp build && gulp watch` should run successfully and http://localhost:5000/ should not look broken.
- `gulp lint` should have less warnings than master.

### CODE REVIEW

**Comment by anselmbradford** [comment]
@contolini

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +0,0 @@
...
-  - cp test/test-runner.js dist/test-runner.js
-  - cp -r test/sample_data/ dist/sample_data/
-  - cp -r test/static/ dist/static/
-deploy:
-  provider: pages
-  local_dir: dist
-  skip_cleanup: true
-  github_token: $GITHUB_GHPAGES_TOKEN
-  on:
-    branch: master
```
Comment (anselmbradford) [inline .travis.yml:23]: Does this need to be migrated to Github Actions? If so, how?

Comment (wpears) [reply .travis.yml:23]: The background is that it [keeps the gh-pages branch in sync](https://github.com/cfpb/cfpb-chart-builder/pull/67), so seems like we'd use [this action](https://github.com/marketplace/actions/deploy-to-github-pages)

Comment (anselmbradford) [reply .travis.yml:23]: Oh hmm, I wonder if I should leave it off for now anyway, considering https://cfpb.github.io/cfpb-chart-builder/ is broken (see https://github.com/cfpb/cfpb-chart-builder/issues/113). I can leave a note in 113 to add it to GitHub Actions.

Comment (wpears) [reply .travis.yml:23]: I'm okay with that, in which case :+1: overall

**Pull request merged**